### PR TITLE
Fix logging on TokenManager

### PR DESCRIPTION
**Pull request by guperrot**
Please have a look at our [guidelines for contributions](https://github.com/Microsoft/AppCenter-SDK-Android/blob/develop/CONTRIBUTING.md) and consider the following before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.

## Description

Remove warning of token not found if token is found...

## Related PRs or issues

[AB#60022](https://msmobilecenter.visualstudio.com/web/wi.aspx?pcguid=27bf89de-9697-418a-accf-1aa125b22914&id=60022)

### CODE REVIEW

File: sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java [hunk 1]
```diff
@@ -61,21 +62,23 @@ TokenResult getCachedToken(String partitionName) {
...
-            Calendar utcCalendar = Calendar.getInstance(TimeZone.getTimeZone("GMT"));
+        if (token != null) {
+            if (!includeExpiredToken) {
 
-            /* The token is expired. */
-            if (utcCalendar.getTime().compareTo(token.expiresOn()) > 0) {
-                AppCenterLog.warn(LOG_TAG, String.format("Cached token result is expired for partition '%s'", partitionName));
-                return null;
+                /* If the token is expired. */
+                Calendar utcCalendar = Calendar.getInstance(TimeZone.getTimeZone("GMT"));
```
Comment (dhei) [inline sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: Getting "GMT" timezone calendar and name it `utcCalendar`?

Comment (guperrot) [reply sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: Not my change and planning to get rid of Calendar here.

File: sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java [hunk 2]
```diff
@@ -61,21 +60,22 @@ TokenResult getCachedToken(String partitionName) {
...
-            Calendar utcCalendar = Calendar.getInstance(TimeZone.getTimeZone("GMT"));
+        if (token != null) {
+            if (!includeExpiredToken) {
 
-            /* The token is expired. */
-            if (utcCalendar.getTime().compareTo(token.expiresOn()) > 0) {
-                AppCenterLog.warn(LOG_TAG, String.format("Cached token result is expired for partition '%s'", partitionName));
-                return null;
+                /* If the token is expired. */
+                if (System.currentTimeMillis() >= token.expiresOn().getTime()) {
```
Comment (YiMysty) [inline sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: is it seconds or milliseconds?...

Comment (YiMysty) [reply sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: nvm seems milli seconds

File: sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java (hunk 1 above)
Comment (YiMysty) [reply sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: well GMT is a timezone, this is the difference.
https://www.timeanddate.com/time/gmt-utc-time.html

File: sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java (hunk 2 above)
Comment (guperrot) [reply sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: Millis

Comment (guperrot) [reply sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: https://developer.android.com/reference/java/util/Date.html#getTime()

Comment (guperrot) [reply sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java]: https://developer.android.com/reference/java/lang/System.html#currentTimeMillis() => already UTC in millis.

Also tested end 2 end and check with debugger on a PST device.

**Pull request merged**